### PR TITLE
New package: cryptomator-cli-0.4.0

### PR DESCRIPTION
**Pull request by nielznl**
I use this program myself, let me know whether it fits in the packages collection

### CODE REVIEW

File: srcpkgs/cryptomator-cli/template [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	echo '#!/bin/sh
+	MYSELF=$(which "$0" 2>/dev/null)
+	[ $? -gt 0 -a -f "$0" ] && MYSELF="./$0"
+	java=java
+	if test -n "$JAVA_HOME"; then
+	    java="$JAVA_HOME/bin/java"
+	fi
+	exec "$java" $java_args -jar $MYSELF "$@"
+	exit 1' > stub.sh
+	cat stub.sh "${pkgname}-${version}.jar" > ${pkgname}
```
Comment (ericonr) [inline srcpkgs/cryptomator-cli/template:31]: Assuming I understood this correctly, you shouldn't do this, the preferred solution is to make a wrapper script in `files/` and `exec java -jar` in it.

Also, `which` isn't installed by default, you can just use `command -v`

File: srcpkgs/cryptomator-cli/template [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Template file for 'cryptomator-cli'
+pkgname=cryptomator-cli
+version=0.4.0
+revision=1
+wrksrc="cli-${version}"
+makedepends="apache-maven"
+depends="openjdk11 davfs2"
```
Comment (ericonr) [inline srcpkgs/cryptomator-cli/template:7]: Is it really openjdk11 or does it work with an earlier one?

**Review by ericonr — COMMENTED** [review]
Remember to squash your commits into a single one :)

File: srcpkgs/cryptomator-cli/template (hunk 2 above)
Comment (nielznl) [reply srcpkgs/cryptomator-cli/template:7]: The readme says 'at least JDK 11'. I tried running it with JDK 8 but it is compiled for JDK 11.

File: srcpkgs/cryptomator-cli/template (hunk 1 above)
Comment (nielznl) [reply srcpkgs/cryptomator-cli/template]: Yeah I was thinking about something like that. I created the wrapper script to be put in /bin and I am now copying the jar file to /usr/share/${pkgname}.

Comment (nielznl) [reply srcpkgs/cryptomator-cli/template]: I thought /usr/share/${pkgname} would be the right location

**Pull request closed without merging**